foundations/foundation-registry/src/main/java/org/apache/servicecomb/registry/api/registry/MicroserviceInstance.java

```diff
@@ -188,6 +189,9 @@ public static MicroserviceInstance createFromDefinition(Configuration configurat
 
     // load properties
     Map<String, String> propertiesMap = InstancePropertiesLoader.INSTANCE.loadProperties(configuration);
+
+    // add register time for warm up
+    propertiesMap.put(InstancePropertiesConst.REGISTER_TIME_KEY, String.valueOf(System.currentTimeMillis()));
```

Comment:
Distributed time may not always same for different instance.

Reply:
add provider invoke time to calculate weight